pkg/codegen/python/gen.go

```diff
@@ -378,7 +378,7 @@ func (mod *modContext) genAwaitableType(w io.Writer, obj *schema.ObjectType) str
 	awaitableName := "Awaitable" + baseName
 
 	// Produce an awaitable subclass.
-	fmt.Fprintf(w, "class %s(%s):\n", awaitableName, baseName)
+	fmt.Fprintf(w, "\n\nclass %s(%s):\n", awaitableName, baseName)
```

Comment:
Nit: Instead of the `\n\n` before `class` what do you think of doing something like the following?

```go
}
fmt.Fprintf(w, "\n\n")

awaitableName := "Awaitable" + baseName

// Produce an awaitable subclass.
fmt.Fprintf(w, "class %s(%s):\n", awaitableName, baseName)
```

Reply:
I agree with moving `fmt.Fprintf(w, "\n\n")` to a separate line but think it should be under the `// Produce an awaitable subclass.` comment since I'd consider the newlines part of that subclass definition and they wouldn't be there without it.